tests/test_pipeline.py

```diff
@@ -216,6 +273,16 @@ def test_serialize_deserialize_pubsub(self, my_pipeline_pubsub, my_entity):
             if isinstance(my_pipeline_pubsub, ParametrizedPipeline):
                 assert loaded.params  # ensure the params area loaded
 
+    def test_serialize_deserialize_env(self, my_pipeline_env, my_entity):
```

Comment:
I don't really see the point of this new test.
Aren't you supposed to also test that the new schedule format does serialize and serialize properly ?

Reply:
This test just show that the Pipeline object (with env schedule defined) can correctly being created & can be deserialized without breaking everything. But actually the **test_init_env** & **test_to_dict_env** is already testing this. Should I remove it ?